### PR TITLE
update Debezium doc with source config

### PR DESCRIPTION
**Pull request by taroface**
@Jeremyyang920, let me know if there are any more fields I should substitute with placeholders.

### CODE REVIEW

File: src/current/v23.2/debezium.md [hunk 1]
```diff
@@ -56,32 +56,60 @@ Once all of the [prerequisite steps](#before-you-begin) are completed, you can u
...
+        "auto.evolve": false,
+        "transforms": "unwrap",
+        "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
+      }
     }
     ~~~
     
-    Specify the **Connection URL** in [JDBC format]({% link {{ page.version.version }}/connect-to-the-database.md %}?filters=java&#step-5-connect-to-the-cluster). For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster]({% link {{ page.version.version }}/connect-to-the-database.md %}).
+    - Specify `connection.url` in [JDBC format]({% link {{ page.version.version }}/connect-to-the-database.md %}?filters=java&#step-5-connect-to-the-cluster). For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster]({% link {{ page.version.version }}/connect-to-the-database.md %}).
+    - If you set `insert.mode` to `upsert`, you must set `pk.mode` and `pk.fields`. Set `pk.fields` to your primary key.
```
Comment (Jeremyyang920) [inline src/current/v23.2/debezium.md:112]: I don't actually recall if this statement is true.... I think maybe if we just link the docs of possible connector options is good enough and we call out everything here is just an example and modify to your needs?

Comment (taroface) [reply src/current/v23.2/debezium.md]: Updated, PTAL!

File: src/current/v23.1/debezium.md [hunk 2]
```diff
@@ -56,33 +56,62 @@ Once all of the [prerequisite steps](#before-you-begin) are completed, you can u
        curl -sO https://packages.confluent.io/maven/io/confluent/kafka-connect-jdbc/$KAFKA_JDBC_VERSION/kafka-connect-jdbc-$KAFKA_JDBC_VERSION.jar
     ~~~
 
-1. Create the JSON configuration file that you will use to create the sink:
+1. Creat the JSON configuration file that you will use to add data from your [source database](https://debezium.io/documentation/reference/stable/connectors/index.html) to a Kafka topic. For example:
```
Comment (Jeremyyang920) [inline src/current/v23.1/debezium.md:59]:
```suggestion
1. Create the JSON configuration file that you will use to add data from your [source database](https://debezium.io/documentation/reference/stable/connectors/index.html) to a Kafka topic. For example:
```

File: src/current/v23.2/debezium.md [hunk 3]
```diff
@@ -56,32 +56,61 @@ Once all of the [prerequisite steps](#before-you-begin) are completed, you can u
        curl -sO https://packages.confluent.io/maven/io/confluent/kafka-connect-jdbc/$KAFKA_JDBC_VERSION/kafka-connect-jdbc-$KAFKA_JDBC_VERSION.jar
     ~~~
 
-1. Create the JSON configuration file that you will use to create the sink:
+1. Creat the JSON configuration file that you will use to add data from your [source database](https://debezium.io/documentation/reference/stable/connectors/index.html) to a Kafka topic. For example:
```
Comment (Jeremyyang920) [inline src/current/v23.2/debezium.md:59]:
```suggestion
1. Create the JSON configuration file that you will use to add data from your [source database](https://debezium.io/documentation/reference/stable/connectors/index.html) to a Kafka topic. For example:
```

**Review by Jeremyyang920 — APPROVED** [review]
LGTM just the typo on the word "create"

**Review by Amruta-Ranade — APPROVED** [review]

**Pull request merged**